### ISSUE TITLE
NuGet Installer Crash on Launch

### ISSUE THREAD

**Issue by AndhikaWB** [sandboxie-plus/Sandboxie#82]
Some (if not all) NuGet installers like WhatsApp and GreenTunnel will crash immediately on launch (silent crash). Sandboxie v5.41.2 x64. This is the error message: `SBIE2224 Sandboxed program has crashed: WhatsAppSetup.exe [DefaultBox]`. If I remember correctly, they will also crash on the last official version of Sandboxie.

**Comment by NewKidOnTheBlock** [comment]
Is that still the case?
Could you post a download link?

**Comment by AndhikaWB** [comment]
@NewKidOnTheBlock Tested using Sandboxie Classic 5.46.5, it works but only if it was run without administrator access. When using `Run As UAC Administrator` it shows the same error dialog. Now I think the older 5.41.2 also share the same behavior (crash when being run as admin).

But if you think this shouldn't be an issue then you can close it.

**Comment by NewKidOnTheBlock** [comment]
Huh?
Why would you want to run it with admin rights if it works fine without?
I gotta say, I haven't checked that box on any program I've tried to run in a sandbox ever.

**Comment by AndhikaWB** [comment]
Sometimes I make [portable apps](https://portableapps.com/) on my free time. I detect data & registry leftovers using Sandboxie. Executing installer as admin usually bring more leftovers (e.g. file association, machine-wide settings) compared to normal user. Based on my experience some apps will adjust the privileges (as admin) automatically when Sandboxed, but not all. It's about preferences.

But yes, as a normal user too (when not making portable apps), I agree that granting admin access is somewhat useless as long as the app works fine without it (and as my previous statement, some apps adjust it automatically). And since I already got what I want not long after creating this issue (I actually forgot I created it), I think I will just close it for now.

Thank you for your time and effort, I appreciate it.
